src/server.js

```diff
@@ -248,8 +243,7 @@ function init(socket, client) {
 								const obj = {};
 
 								if (success) {
-									obj.success = "Successfully updated your password, all your other sessions were logged out";
-									obj.token = client.config.token;
+									obj.success = "Successfully updated your password";
```

Comment:
I tried to find what's the internet's opinion on this, and can't find anything conclusive. Google seems to be keeping sessions on password change?

If I were to add session destroying back, there's the real consequence of killing push notifications (that PR depends on this PR) when changing the password. Do we want that?